chart/values.yaml

```diff
@@ -2,13 +2,13 @@
 # Licensed under the MIT License. See License-MIT.txt in the project root for license information.
 
 # version of Gitpod will be set by the CI job on publishing to https://charts.gitpod.io/
-version:
+version: 0.8.1
 
 hostname: localhost
 certificatesSecret:
   secretName: https-certificates
 
-imagePrefix: gcr.io/gitpod-io/self-hosted/
+imagePrefix: public.ecr.aws/b1o4t7o5/
```

Comment:
This gets overwrriten by prod or stage, dont change this

Reply:
this value will be updated by CI/CD